warehouse-management-service/internal/handler/handler.go

```diff
@@ -0,0 +1,21 @@
+package handler
+
+import (
+	"net/http"
+	"warehouse-management-service/pkg/database"
+)
+
+type handler struct {
+	db database.Service
+}
+
+func New(db database.Service) http.Handler {
+	handler := &handler{
+		db: db,
+	}
+	return handler.router()
+}
+
+func (h *handler) Ping(w http.ResponseWriter, r *http.Request) {
+	w.Write([]byte("pong"))
```

Comment:
issue (non-blocking): This line swallows the error value returned by the `Write` method.

Let's instead check the error value and log it if the write failed.

Reply:
Added err checking and a logging here